src/components/UserProfile/UserProfile.module.scss

```
@@ -0,0 +1,23 @@
+.userProfile {
```

```suggestion
.user-profile {
```
css 네이밍 스타일을 변경하는 것은 어떨까요??

---

저 파일이 module.scss 라서 하이픈은 한 단어로 인식을 하지 못하더라구요~
그래서 카멜 케이스를 적용하였습니다!

---

진옥님 말씀이 맞습니다

---

오 그렇군요! 알겠습니다! 굳굳이요! 👍